plugin/advisor/sql_review.go

```diff
@@ -95,6 +95,8 @@ const (
 	SchemaRuleIndexNoDuplicateColumn SQLReviewRuleType = "index.no-duplicate-column"
 	// SchemaRuleIndexKeyNumberLimit enforce the index key number limit.
 	SchemaRuleIndexKeyNumberLimit SQLReviewRuleType = "index.key-number-limit"
+	// SchemaRuleIndexUnexpectedPKType require expected type of primary key.
+	SchemaRuleIndexUnexpectedPKType SQLReviewRuleType = "index.unexpected-pk-type"
```

Comment:
`UnexpectedPKType` sounds like an error.
```suggestion
	// SchemaRuleIndexPKType enforce the type restriction of columns in primary key.
	SchemaRuleIndexPKType SQLReviewRuleType = "index.pk-type"
```